website/coursework/product-design/index.md

```diff
@@ -103,12 +122,15 @@ List the above in a _user persona_ card. A persona is a fictional representation
 
 ![Example Persona](img/persona.jpg)
 
-<small>Source: <https://uxdesign.cc/meetup-a-usability-case-study-e909c33f1e3e></small>
+<small>Source: https://uxdesign.cc/meetup-a-usability-case-study-e909c33f1e3e</small>
```

Comment:
Hmm should we make these links clickable? I think both are fine, so long we standardise.

Reply:
Yes. Lemme do it in a followup PR where I'll use more semantic ways